commcare_connect/opportunity/tests/test_views.py

```
@@ -1194,3 +1199,191 @@ def test_add_payment_unit_form_with_regular_opportunity(self, client):
         managed_amount_fields = managed_amounts_div.fields
 
         assert len(managed_amount_fields) == 2
+
+
+@pytest.mark.django_db
+class TestInvoiceUpdateStatus:
+    @pytest.fixture
+    def nm_organization(self, db):
```

Ruff (0.14.14)

[warning] Unused method argument: `db`

(ARG002)